### DISCUSSION TITLE
Microsoft.HybridCompute/privateLinkScopes

### DISCUSSION THREAD

**Discussion by Grant-Rc** [Azure/ResourceModules#4527] [General]
Morning all 

Loving the modules as it alot of effort taken care of up front. I was woundering if anyone was working on Microsoft.HybridCompute/privateLinkScopes module? This is for the deployments of ARC and i dont seem to see this in any module

**Comment by eriqua** [comment]
Hi @Grant-Rc , thanks for opening this discussion and again sorry for the late reply.

CARML evolved to and has been rebranded as the Bicep version of Azure Verified Modules ([AVM](https://aka.ms/AVM)). 

All modules previously hosted by this repo have been updated to align with AVM specifications and published to the Public Bicep Registry. Their evolved source code can be found in the underlying repository ([BRM](https://aka.ms/BRM)).

However, in order not to lose relevant discussions like this one, all CARML GitHub discussions are being converted to CARML GitHub issues, in order for the team to be able to further triage them and migrate them to AVM issues, if applicable.

Please refer to issue #4548 going forward.
